bcipy/gui/mode/RSVPKeyboard.py

```diff
@@ -15,7 +18,12 @@
 class RSVPKeyboard(BCIGui):
     """GUI for launching the RSVP tasks."""
     event_started = False
-    PARAMETER_LOCATION = 'bcipy/parameters/parameters.json'
+
+    def __init__(self, title, size, background_color, default_param_location='bcipy/parameters/parameters.json',
+                 params_queue=None):
+        super(RSVPKeyboard, self).__init__(title, size, background_color)
+        self.PARAMETER_LOCATION = default_param_location
```

Comment:
I like this design, it's a lot more flexible yet leaves the original functionality as a default!